mlfinlab/docs/source/implementations/clustering.rst

```diff
@@ -0,0 +1,117 @@
+.. _implementations-clustering:
```

Comment:
Why are there two files called clustering? I'm confused, how was it even possible to commit 2 files in the same dir with the same name?

Reply:
Hmm please double check. There are 8 files here and 4 of them are duplicates but not quite.